src/main/webapp/buttonscript.js

```diff
@@ -1,165 +1,164 @@
 
 
 function GetSch() {
-          const leapworkHostname = document.getElementById("leapworkHostname").value;
-          const leapworkPort = document.getElementById("leapworkPort").value;
-
-          if(!leapworkHostname || !leapworkPort)
-          {
-            alert('"hostname or/and field is empty! Cannot connect to controller!"');
-          }
-          else
-          {
-              const address = "http://" + leapworkHostname + ":" + leapworkPort;
-              const accessKey = document.getElementById("leapworkAccessKey").value;
-
-              if(document.getElementById('LeapworkContainer').innerHTML == "")
-              {
+	var closestEle =  $(event.target).closest('.repeated-chunk').firstElementChild;
```

Comment:
It could be stop working if class name will get changes in Jenkins, in order to make it safe try to verify the exitance of  the element and if it not exist than use document as root element.

Reply:
Updated the code in latest commit